pkg/api/index.go

```diff
@@ -77,6 +77,16 @@ func SearchIndexHandler(params index.SearchIndexParams) middleware.Responder {
 		}
 		result = append(result, resultUUIDs...)
 	}
+	if params.Query.Email != "" {
+		if !govalidator.IsEmail(params.Query.Email) {
```

Comment:
we're using different validator packages in client VS server; was this intentional?

Reply:
I went with the already used one in this file. I'll make it uniform:
https://github.com/go-playground/validator to check sha would be a length check (rather than a `IsSHA256` check) and both have email. I would prefer to use this one, WDYT?

Reply:
https://github.com/go-playground/validator has cross-field validation... sweet! I like that one better too.